### PR TITLE
ToggleSwitch

### PR DESCRIPTION
**Pull request by JamRemco**
## What does the pull request do?

WIP for the Toggleswitch control
image

## What is the current behavior?

There is ToggleSwitch control yet.

## What is the updated/expected behavior with this PR?

For now its a dressed up Togglebutton (like checkbox)

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

File: src/Avalonia.Themes.Default/ToggleSwitch.xaml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+   <Style Selector="ToggleSwitch:pressed  /template/ Border#PART_BorderBackground">
+    <Setter Property="BorderBrush" Value="{DynamicResource ThemeControlHighBrush}"/>
+  </Style>
+ 
+      <Style Selector="ToggleSwitch:checked /template/ ContentControl#PART_TextBooleanValue">
+        <Setter Property="Content" Value="On"/>
+      </Style>
+
+      <Style Selector="ToggleSwitch:unchecked /template/ ContentControl#PART_TextBooleanValue">
+        <Setter Property="Content" Value="Off"/>
```
Comment (maxkatz6) [inline src/Avalonia.Themes.Default/ToggleSwitch.xaml:148]: I think, Off and On values should be moved to property from the start?
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.toggleswitch.offcontent?view=winrt-19041
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.toggleswitch.oncontent?view=winrt-19041

Comment (JamRemco) [reply src/Avalonia.Themes.Default/ToggleSwitch.xaml:148]: @maxkatz6 Yes, in the future the control wil evolve to a more flexibel control.

Comment (JamRemco) [reply src/Avalonia.Themes.Default/ToggleSwitch.xaml]: Toggle

**Review by danwalmsley — APPROVED** [review]
LGTM well done @JamRemco

**Pull request merged**